### PR TITLE
Completed About Page with Functional Nav Bar

### PR DESCRIPTION
**Pull request by fadiismail27**
Added styling with the navbar similar to the figma file and finished the about page.

### CODE REVIEW

**Review by jasozh — CHANGES_REQUESTED** [review]
Navbar link doesn't go to the correct URLs. The code itself also needs to be formatted with Prettier correctly.

File: src/app/about/page.tsx [hunk 1]
```diff
@@ -1,5 +1,92 @@
...
+
+
+      {/* Main Content Section Only */}
+      <div className="flex flex-col md:flex-row items-center justify-center max-w-6xl mx-auto mt-12 p-8">
+        {/* Avatar Section */}
+        <div className="flex-shrink-0 mb-6 md:mb-0 md:mr-12">
+          <Image
+            src="/image.png" 
+            alt="Martha Pollack"
+            className="rectangle"
```
Comment (juliapapp24) [inline src/app/about/page.tsx:65]: "rectangle" is not the best name for an image class, consider changing it to "profile-pic-container" or something like that

File: src/app/about/page.tsx [hunk 2]
```diff
@@ -1,5 +1,92 @@
...
+            width={400}
+            height={500}
```
Comment (juliapapp24) [inline src/app/about/page.tsx:67]: consider adding "layout="responsive" to make dynamically resize the image based on the screen size

**Review by jasozh — CHANGES_REQUESTED** [review]
You should not be pushing any changes to yarn.lock, package.json, or package-lock.json.

**Review by jasozh — APPROVED** [review]

File: src/app/about/page.tsx [hunk 3]
```diff
@@ -1,5 +1,95 @@
...
 const About = () => {
-  return <div>Hello</div>;
+  return (
+    <main className="min-h-screen">
+      {/* Navbar */}
+      <nav className="bg-white shadow-md">
+        <div className="flex flex-col items-center bg-white p-6 md:flex-row md:items-center md:justify-between">
+          {/* Title */}
+          <h1 className="mb-4 text-3xl font-bold text-red-600 md:mb-0">
+            Martha Pollack's Portfolio
```
Comment (juliapapp24) [inline src/app/about/page.tsx:11]: fix ' to correct thing

File: src/app/about/page.tsx [hunk 4]
```diff
@@ -1,5 +1,95 @@
+import Image from "next/image";
+
 const About = () => {
-  return <div>Hello</div>;
+  return (
+    <main className="min-h-screen">
+      {/* Navbar */}
+      <nav className="bg-white shadow-md">
```
Comment (juliapapp24) [inline src/app/about/page.tsx:7]: remove background from this and make it transparent like it is on figma

**Pull request merged**